drivers/i2s/i2s_sam_ssc.c

```diff
@@ -631,13 +620,11 @@ static int i2s_sam_configure(const struct device *dev, enum i2s_dir dir,
 	return 0;
 }
 
-static int rx_stream_start(struct stream *stream, Ssc *const ssc,
-			   const struct device *dev_dma)
+static int rx_stream_start(struct stream *stream, Ssc *const ssc, const struct device *dev_dma)
```

Comment:
Argument name can't be the same as the struct name

```suggestion
static int rx_stream_start(struct stream *str, Ssc *const ssc, const struct device *dev_dma)
```